docs/ref/contrib/contenttypes.txt

```diff
@@ -106,14 +106,19 @@ methods that allow you to get from a
 :class:`~django.contrib.contenttypes.models.ContentType` instance to the
 model it represents, or to retrieve objects from that model:
 
-.. method:: ContentType.get_object_for_this_type(**kwargs)
+.. method:: ContentType.get_object_for_this_type(using=None, **kwargs)
 
     Takes a set of valid :ref:`lookup arguments <field-lookups-intro>` for the
     model the :class:`~django.contrib.contenttypes.models.ContentType`
     represents, and does
     :meth:`a get() lookup <django.db.models.query.QuerySet.get>`
     on that model, returning the corresponding object.
 
+    .. versionchanged:: 5.1
+
+        The ``using`` argument was added, for specifying a different database
+        than the default.
```

Comment:
```suggestion
    .. versionchanged:: 5.1

        The ``using`` argument was added.
```

We can add an explanation how it works to the main `get_object_for_this_type()` docs.

Reply:
I updated the docs.